demos/evp/Makefile

```diff
@@ -18,3 +18,6 @@ aesgcm: aesgcm.o
 
 aesccm aesgcm:
 	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $<
+
+clean:
+	$(RM) aesccm.o aesgcm.o aesccm aesgcm
```

Comment:
No I use vim and cscope on Linux and/or Solaris. 
You mean add a newline like this ?

$git diff Makefile
diff --git a/demos/evp/Makefile b/demos/evp/Makefile
index ebf5cf1..ffa8acc 100644
--- a/demos/evp/Makefile
+++ b/demos/evp/Makefile
@@ -21,3 +21,4 @@ aesccm aesgcm:
 
 clean:
        $(RM) aesccm.o aesgcm.o aesccm aesgcm
+